# Data Visualisation Statistiques descriptives

## Formaliser des KPI

#### 1) Se renseigner sur ce qu'est un KPI

Un KPI ou Key Performance Indicator (Indicateur clé de performance) est un élément chiffré permettant l'analyse et la mesure de l'efficacité d'une campagne marketing.

Cela donne à l'entreprise l'idée d'un objectif à atteindre et d'observer la tendance pour réussir cette objectif par l'élaboration ou le réajustement de leur stratégie marketing.

Ces KPIs sont suivis quotidiennement et compilés dans des tableaux de bord, appelés reportings.

#### 2) Importez les bonnes librairies et le dataset chipotle.csv et afficher les infos du dataset

In [90]:
import pandas as pd
import seaborn as sns
import matplotlib as mp
import matplotlib.pyplot as plt
import numpy as np

print("lib loaded")

lib loaded


In [91]:
dataset = pd.read_csv("./chipotle.csv", sep=",")

print("dataset loaded")

dataset.head()

dataset loaded


Unnamed: 0  order id  quantity                              item_name  \
0           0         1         1           Chips and Fresh Tomato Salsa   
1           1         1         1                                   Izze   
2           2         1         1                       Nantucket Nectar   
3           3         1         1  Chips and Tomatillo-Green Chili Salsa   
4           4         2         2                           Chicken Bowl   

                                  choice_description item_price  
0                                                NaN     $2.39   
1                                       [Clementine]     $3.39   
2                                            [Apple]     $3.39   
3                                                NaN     $2.39   
4  [Tomatillo-Red Chili Salsa (Hot), [Black Beans...    $16.98

In [92]:
dataset.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 4622 entries, 0 to 4621
Data columns (total 6 columns):
 #   Column              Non-Null Count  Dtype 
---  ------              --------------  ----- 
 0   Unnamed: 0          4622 non-null   int64 
 1   order id            4622 non-null   int64 
 2   quantity            4622 non-null   int64 
 3   item_name           4622 non-null   object
 4   choice_description  3376 non-null   object
 5   item_price          4622 non-null   object
dtypes: int64(3), object(3)
memory usage: 216.8+ KB


#### 3) Regardez les 10 premières lignes du dataset

In [93]:
dataset.head(10)

Unnamed: 0  order id  quantity                              item_name  \
0           0         1         1           Chips and Fresh Tomato Salsa   
1           1         1         1                                   Izze   
2           2         1         1                       Nantucket Nectar   
3           3         1         1  Chips and Tomatillo-Green Chili Salsa   
4           4         2         2                           Chicken Bowl   
5           5         3         1                           Chicken Bowl   
6           6         3         1                          Side of Chips   
7           7         4         1                          Steak Burrito   
8           8         4         1                       Steak Soft Tacos   
9           9         5         1                          Steak Burrito   

                                  choice_description item_price  
0                                                NaN     $2.39   
1                                       [Clementine]     $3.39   
2                                            [Apple]     $3.39   
3                                                NaN     $2.39   
4  [Tomatillo-Red Chili Salsa (Hot), [Black Beans...    $16.98   
5  [Fresh Tomato Salsa (Mild), [Rice, Cheese, Sou...    $10.98   
6                                                NaN     $1.69   
7  [Tomatillo Red Chili Salsa, [Fajita Vegetables...    $11.75   
8  [Tomatillo Green Chili Salsa, [Pinto Beans, Ch...     $9.25   
9  [Fresh Tomato Salsa, [Rice, Black Beans, Pinto...     $9.25

In [94]:
dataset.iloc[:10, :]

Unnamed: 0  order id  quantity                              item_name  \
0           0         1         1           Chips and Fresh Tomato Salsa   
1           1         1         1                                   Izze   
2           2         1         1                       Nantucket Nectar   
3           3         1         1  Chips and Tomatillo-Green Chili Salsa   
4           4         2         2                           Chicken Bowl   
5           5         3         1                           Chicken Bowl   
6           6         3         1                          Side of Chips   
7           7         4         1                          Steak Burrito   
8           8         4         1                       Steak Soft Tacos   
9           9         5         1                          Steak Burrito   

                                  choice_description item_price  
0                                                NaN     $2.39   
1                                       [Clementine]     $3.39   
2                                            [Apple]     $3.39   
3                                                NaN     $2.39   
4  [Tomatillo-Red Chili Salsa (Hot), [Black Beans...    $16.98   
5  [Fresh Tomato Salsa (Mild), [Rice, Cheese, Sou...    $10.98   
6                                                NaN     $1.69   
7  [Tomatillo Red Chili Salsa, [Fajita Vegetables...    $11.75   
8  [Tomatillo Green Chili Salsa, [Pinto Beans, Ch...     $9.25   
9  [Fresh Tomato Salsa, [Rice, Black Beans, Pinto...     $9.25

#### 4) Quelle est la taille de notre dataset ?

In [95]:
dataset.shape #(lignes, colonnes)

(4622, 6)

#### 5) Affichez toutes les colonnes de notre dataset

In [96]:
dataset.columns

Index(['Unnamed: 0', 'order id', 'quantity', 'item_name', 'choice_description',
       'item_price'],
      dtype='object')

#### 6) Supprimez les colonnes choice_description et Unnamed: 0

In [97]:
dataset = dataset.drop(columns =['choice_description', 'Unnamed: 0'])

dataset.columns

Index(['order id', 'quantity', 'item_name', 'item_price'], dtype='object')

#### 7) Renommez la colonne order id en ID

In [98]:
dataset.rename(columns={'order id':'ID'}, inplace=True)

dataset.columns

Index(['ID', 'quantity', 'item_name', 'item_price'], dtype='object')

#### 8) Réaffichez les 10 premières lignes

In [99]:
dataset.head(10)

ID  quantity                              item_name item_price
0   1         1           Chips and Fresh Tomato Salsa     $2.39 
1   1         1                                   Izze     $3.39 
2   1         1                       Nantucket Nectar     $3.39 
3   1         1  Chips and Tomatillo-Green Chili Salsa     $2.39 
4   2         2                           Chicken Bowl    $16.98 
5   3         1                           Chicken Bowl    $10.98 
6   3         1                          Side of Chips     $1.69 
7   4         1                          Steak Burrito    $11.75 
8   4         1                       Steak Soft Tacos     $9.25 
9   5         1                          Steak Burrito     $9.25

#### 9) Quel est l'item le plus commandé ?

In [100]:
popular = dataset.groupby('item_name')

print(popular.groups)

{'6 Pack Soft Drink': Int64Index([ 298,  341,  357,  388,  417,  553,  639,  721,  743,  879, 1026,
            1051, 1124, 1167, 1260, 1279, 1392, 1755, 1758, 1849, 1860, 1916,
            1922, 1937, 1976, 2108, 2232, 2374, 2389, 2713, 2793, 2860, 3010,
            3132, 3141, 3389, 3422, 3600, 3614, 3633, 3693, 3766, 3789, 3806,
            3836, 3875, 3886, 4169, 4174, 4359, 4363, 4522, 4527, 4535],
           dtype='int64'), 'Barbacoa Bowl': Int64Index([  39,   58,   95,  115,  127,  136,  219,  254,  418,  471,  592,
             750,  803,  804,  840,  919, 1032, 1068, 1174, 1175, 1179, 1235,
            1236, 1264, 1347, 1357, 1383, 1417, 1419, 1420, 1643, 1650, 1670,
            1746, 1804, 1970, 2013, 2073, 2253, 2372, 2399, 2517, 2620, 2667,
            2749, 2762, 2800, 3017, 3025, 3195, 3245, 3253, 3376, 3394, 3547,
            3549, 3822, 4046, 4056, 4135, 4140, 4380, 4441, 4485, 4510, 4594],
           dtype='int64'), 'Barbacoa Burrito': Int64Index([  21,   81,  121,  13

In [108]:
dicoQuantity = dict() #Quantité totale de chaque item commandé

for item, group in popular:
    dicoQuantity[item] = group['quantity'].sum()
    print(item)
    print(group)
    print("Quantité totale achetée =", group['quantity'].sum())
    print("--------------------------------------")

6 Pack Soft Drink
        ID  quantity          item_name item_price
298    129         1  6 Pack Soft Drink     $6.49 
341    148         1  6 Pack Soft Drink     $6.49 
357    154         1  6 Pack Soft Drink     $6.49 
388    168         1  6 Pack Soft Drink     $6.49 
417    182         1  6 Pack Soft Drink     $6.49 
553    230         1  6 Pack Soft Drink     $6.49 
639    264         1  6 Pack Soft Drink     $6.49 
721    298         1  6 Pack Soft Drink     $6.49 
743    306         1  6 Pack Soft Drink     $6.49 
879    363         1  6 Pack Soft Drink     $6.49 
1026   422         1  6 Pack Soft Drink     $6.49 
1051   432         1  6 Pack Soft Drink     $6.49 
1124   465         1  6 Pack Soft Drink     $6.49 
1167   481         1  6 Pack Soft Drink     $6.49 
1260   511         1  6 Pack Soft Drink     $6.49 
1279   520         1  6 Pack Soft Drink     $6.49 
1392   566         1  6 Pack Soft Drink     $6.49 
1755   708         1  6 Pack Soft Drink     $6.49 
1758   709   

In [102]:
dicoQuantity

{'6 Pack Soft Drink': 55,
 'Barbacoa Bowl': 66,
 'Barbacoa Burrito': 91,
 'Barbacoa Crispy Tacos': 12,
 'Barbacoa Salad Bowl': 10,
 'Barbacoa Soft Tacos': 25,
 'Bottled Water': 211,
 'Bowl': 4,
 'Burrito': 6,
 'Canned Soda': 126,
 'Canned Soft Drink': 351,
 'Carnitas Bowl': 71,
 'Carnitas Burrito': 60,
 'Carnitas Crispy Tacos': 8,
 'Carnitas Salad': 1,
 'Carnitas Salad Bowl': 6,
 'Carnitas Soft Tacos': 40,
 'Chicken Bowl': 761,
 'Chicken Burrito': 591,
 'Chicken Crispy Tacos': 50,
 'Chicken Salad': 9,
 'Chicken Salad Bowl': 123,
 'Chicken Soft Tacos': 120,
 'Chips': 230,
 'Chips and Fresh Tomato Salsa': 130,
 'Chips and Guacamole': 506,
 'Chips and Mild Fresh Tomato Salsa': 1,
 'Chips and Roasted Chili Corn Salsa': 23,
 'Chips and Roasted Chili-Corn Salsa': 18,
 'Chips and Tomatillo Green Chili Salsa': 45,
 'Chips and Tomatillo Red Chili Salsa': 50,
 'Chips and Tomatillo-Green Chili Salsa': 33,
 'Chips and Tomatillo-Red Chili Salsa': 25,
 'Crispy Tacos': 2,
 'Izze': 20,
 'Nantucket Nec

L'item le plus commandé (en quantité) est donc le 'Chicken Bowl' avec 761. 

#### 10) Combien d'items ont été commandés au total chez Chipotle ?

In [103]:
print("Au total, chez Chipotle,", sum(dicoQuantity.values()), "items ont été commandés.")

Au total, chez Chipotle, 4972 items ont été commandés.


#### 11) Combien de chiffre d'affaire a réalisé Chipotle ?

In [104]:
prix = dataset['item_price'].apply(lambda x: x[1:]) #Suppression du 1er caractère ($) de chaque ligne de prix
prix = prix.apply(pd.to_numeric) #Convertion des données en numérique pour les additionner

print("Chipotle a réalisé au total un chiffre d'affaire de", prix.sum(), "$.")

Chipotle a réalisé au total un chiffre d'affaire de 34500.16 $.


#### 12) Quel est le chiffre d'affaire moyen par commande ?

In [109]:
df = dataset.groupby('ID')

print(df.groups)

{1: Int64Index([0, 1, 2, 3], dtype='int64'), 2: Int64Index([4], dtype='int64'), 3: Int64Index([5, 6], dtype='int64'), 4: Int64Index([7, 8], dtype='int64'), 5: Int64Index([9, 10], dtype='int64'), 6: Int64Index([11, 12], dtype='int64'), 7: Int64Index([13, 14], dtype='int64'), 8: Int64Index([15, 16], dtype='int64'), 9: Int64Index([17, 18], dtype='int64'), 10: Int64Index([19, 20], dtype='int64'), 11: Int64Index([21, 22], dtype='int64'), 12: Int64Index([23, 24], dtype='int64'), 13: Int64Index([25, 26], dtype='int64'), 14: Int64Index([27, 28], dtype='int64'), 15: Int64Index([29, 30], dtype='int64'), 16: Int64Index([31, 32], dtype='int64'), 17: Int64Index([33, 34], dtype='int64'), 18: Int64Index([35, 36, 37, 38], dtype='int64'), 19: Int64Index([39, 40], dtype='int64'), 20: Int64Index([41, 42, 43, 44], dtype='int64'), 21: Int64Index([45, 46, 47], dtype='int64'), 22: Int64Index([48, 49], dtype='int64'), 23: Int64Index([50, 51], dtype='int64'), 24: Int64Index([52, 53], dtype='int64'), 25: Int64I

In [106]:
dicoOrder = dict() #Somme totale de chaque commande

for item, group in df:
    dicoOrder[item] = pd.to_numeric(group['item_price'].apply(lambda x: x[1:])).sum()
    print(item)
    print(group)
    print("Somme totale de la commande =", pd.to_numeric(group['item_price'].apply(lambda x: x[1:])).sum())
    print("--------------------------------------")

1
   ID  quantity                              item_name item_price
0   1         1           Chips and Fresh Tomato Salsa     $2.39 
1   1         1                                   Izze     $3.39 
2   1         1                       Nantucket Nectar     $3.39 
3   1         1  Chips and Tomatillo-Green Chili Salsa     $2.39 
Somme totale de la commande = 11.56
--------------------------------------
2
   ID  quantity     item_name item_price
4   2         2  Chicken Bowl    $16.98 
Somme totale de la commande = 16.98
--------------------------------------
3
   ID  quantity      item_name item_price
5   3         1   Chicken Bowl    $10.98 
6   3         1  Side of Chips     $1.69 
Somme totale de la commande = 12.67
--------------------------------------
4
   ID  quantity         item_name item_price
7   4         1     Steak Burrito    $11.75 
8   4         1  Steak Soft Tacos     $9.25 
Somme totale de la commande = 21.0
--------------------------------------
5
    ID  quantity  

153  69         1  Chips and Tomatillo-Green Chili Salsa     $2.39 
Somme totale de la commande = 10.88
--------------------------------------
70
     ID  quantity            item_name item_price
154  70         2         Chicken Bowl    $17.50 
155  70         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 21.95
--------------------------------------
71
     ID  quantity            item_name item_price
156  71         1  Chips and Guacamole     $4.45 
157  71         1          Veggie Bowl    $11.25 
158  71         1        Steak Burrito    $11.75 
Somme totale de la commande = 27.45
--------------------------------------
72
     ID  quantity        item_name item_price
159  72         1  Chicken Burrito    $10.98 
Somme totale de la commande = 10.98
--------------------------------------
73
     ID  quantity        item_name item_price
160  73         1     Chicken Bowl     $8.49 
161  73         1  Chicken Burrito    $10.98 
162  73         2      Canned Soda     $

324  141         1  Chicken Soft Tacos     $8.49 
Somme totale de la commande = 16.98
--------------------------------------
142
      ID  quantity                     item_name item_price
325  142         1                 Steak Burrito     $8.99 
326  142         1  Chips and Fresh Tomato Salsa     $2.39 
Somme totale de la commande = 11.38
--------------------------------------
143
      ID  quantity      item_name item_price
327  143         1   Chicken Bowl     $8.75 
328  143         1          Chips     $2.15 
329  143         1  Bottled Water     $1.50 
Somme totale de la commande = 12.4
--------------------------------------
144
      ID  quantity        item_name item_price
330  144         1  Chicken Burrito    $11.25 
331  144         1            Chips     $2.15 
Somme totale de la commande = 13.4
--------------------------------------
145
      ID  quantity           item_name item_price
332  145         1  Chicken Soft Tacos     $8.49 
333  145         1  Chicken Soft Ta

518  216         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 13.2
--------------------------------------
217
      ID  quantity     item_name item_price
519  217         1       Burrito     $7.40 
520  217         1  Crispy Tacos     $7.40 
521  217         1  Crispy Tacos     $7.40 
Somme totale de la commande = 22.200000000000003
--------------------------------------
218
      ID  quantity            item_name item_price
522  218         1        Steak Burrito     $8.99 
523  218         1  Chips and Guacamole     $3.99 
Somme totale de la commande = 12.98
--------------------------------------
219
      ID  quantity      item_name item_price
524  219         1     Steak Bowl     $8.99 
525  219         1  Side of Chips     $1.69 
Somme totale de la commande = 10.68
--------------------------------------
220
      ID  quantity           item_name item_price
526  220         1  Chicken Salad Bowl     $8.75 
527  220         1        Chicken Bowl     $8.75 
Somme t

709  291         1                   Canned Soft Drink     $1.25 
Somme totale de la commande = 13.45
--------------------------------------
292
      ID  quantity            item_name item_price
710  292         1       Veggie Burrito     $8.49 
711  292         1  Chips and Guacamole     $3.99 
Somme totale de la commande = 12.48
--------------------------------------
293
      ID  quantity      item_name item_price
712  293         1  Steak Burrito    $11.48 
Somme totale de la commande = 11.48
--------------------------------------
294
      ID  quantity            item_name item_price
713  294         1  Chips and Guacamole     $3.99 
714  294         1       Veggie Burrito     $8.49 
Somme totale de la commande = 12.48
--------------------------------------
295
      ID  quantity          item_name item_price
715  295         1    Chicken Burrito    $11.25 
716  295         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 12.5
--------------------------------------
2

925  381         1    Nantucket Nectar     $3.39 
Somme totale de la commande = 20.37
--------------------------------------
382
      ID  quantity            item_name item_price
926  382         1           Steak Bowl     $8.69 
927  382         1  Chips and Guacamole     $3.89 
Somme totale de la commande = 12.58
--------------------------------------
383
      ID  quantity            item_name item_price
928  383         1         Chicken Bowl     $8.75 
929  383         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 13.2
--------------------------------------
384
      ID  quantity       item_name item_price
930  384         1    Chicken Bowl     $8.75 
931  384         1  Veggie Burrito    $11.25 
932  384         1           Chips     $2.15 
933  384         1    Chicken Bowl     $8.75 
Somme totale de la commande = 30.9
--------------------------------------
385
      ID  quantity      item_name item_price
934  385         1   Chicken Bowl     $8.49 
935  385  

1110  459         2  Chicken Salad Bowl    $22.50 
Somme totale de la commande = 22.5
--------------------------------------
460
       ID  quantity            item_name item_price
1111  460         1           Steak Bowl     $9.25 
1112  460         1         Chicken Bowl     $8.75 
1113  460         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 22.45
--------------------------------------
461
       ID  quantity          item_name item_price
1114  461         1      Steak Burrito    $11.75 
1115  461         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 13.0
--------------------------------------
462
       ID  quantity                     item_name item_price
1116  462         1               Chicken Burrito    $11.25 
1117  462         1               Chicken Burrito    $11.25 
1118  462         1  Chips and Fresh Tomato Salsa     $2.95 
Somme totale de la commande = 25.45
--------------------------------------
463
       ID  quantity         item_

1304  531         1  Chips and Guacamole     $3.99 
Somme totale de la commande = 12.48
--------------------------------------
532
       ID  quantity      item_name item_price
1305  532         1  Side of Chips     $1.69 
1306  532         1     Steak Bowl     $8.99 
Somme totale de la commande = 10.68
--------------------------------------
533
       ID  quantity      item_name item_price
1307  533         1   Chicken Bowl     $8.49 
1308  533         1  Side of Chips     $1.69 
Somme totale de la commande = 10.18
--------------------------------------
534
       ID  quantity         item_name item_price
1309  534         1     Steak Burrito    $11.75 
1310  534         1   Chicken Burrito    $11.25 
1311  534         1  Steak Salad Bowl    $11.89 
Somme totale de la commande = 34.89
--------------------------------------
535
       ID  quantity                            item_name item_price
1312  535         1                           Steak Bowl     $9.25 
1313  535         1  Chi

1495  607         1             Canned Soft Drink     $1.25 
Somme totale de la commande = 12.95
--------------------------------------
608
       ID  quantity                     item_name item_price
1496  608         1            Chicken Soft Tacos     $8.49 
1497  608         1                Veggie Burrito     $8.49 
1498  608         1  Chips and Fresh Tomato Salsa     $2.39 
Somme totale de la commande = 19.37
--------------------------------------
609
       ID  quantity          item_name item_price
1499  609         1       Chicken Bowl    $11.25 
1500  609         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 12.5
--------------------------------------
610
       ID  quantity        item_name item_price
1501  610         1  Chicken Burrito     $8.75 
1502  610         1  Chicken Burrito     $8.75 
Somme totale de la commande = 17.5
--------------------------------------
611
       ID  quantity     item_name item_price
1503  611         1    Steak Bowl     $9.2

1676  678         1  Chips and Guacamole     $3.99 
Somme totale de la commande = 12.98
--------------------------------------
679
       ID  quantity         item_name item_price
1677  679         1  Carnitas Burrito    $11.75 
1678  679         1             Chips     $2.15 
Somme totale de la commande = 13.9
--------------------------------------
680
       ID  quantity      item_name item_price
1679  680         1  Carnitas Bowl     $8.99 
1680  680         1           Izze     $3.39 
Somme totale de la commande = 12.38
--------------------------------------
681
       ID  quantity           item_name item_price
1681  681         1       Steak Burrito     $9.25 
1682  681         1  Chicken Soft Tacos     $8.75 
Somme totale de la commande = 18.0
--------------------------------------
682
       ID  quantity            item_name item_price
1683  682         1         Chicken Bowl    $11.25 
1684  682         1         Chicken Bowl    $11.25 
1685  682         1  Chips and Guacamole

1882  759         2  Chips and Fresh Tomato Salsa     $5.90 
Somme totale de la commande = 86.30000000000001
--------------------------------------
760
       ID  quantity           item_name item_price
1883  760         1  Chicken Salad Bowl    $11.25 
1884  760         1   Veggie Salad Bowl    $11.25 
Somme totale de la commande = 22.5
--------------------------------------
761
       ID  quantity        item_name item_price
1885  761         1  Chicken Burrito     $8.49 
1886  761         1  Chicken Burrito    $10.98 
Somme totale de la commande = 19.47
--------------------------------------
762
       ID  quantity      item_name item_price
1887  762         1   Chicken Bowl     $8.75 
1888  762         1          Chips     $2.15 
1889  762         1  Bottled Water     $1.50 
Somme totale de la commande = 12.4
--------------------------------------
763
       ID  quantity          item_name item_price
1890  763         1         Steak Bowl     $9.25 
1891  763         1  Canned Soft

2110  849         1      Chicken Burrito    $11.25 
Somme totale de la commande = 22.19
--------------------------------------
850
       ID  quantity           item_name item_price
2111  850         1  Chicken Salad Bowl    $11.25 
2112  850         2               Chips     $4.30 
2113  850         1  Chicken Salad Bowl     $8.75 
Somme totale de la commande = 24.3
--------------------------------------
851
       ID  quantity            item_name item_price
2114  851         1         Chicken Bowl    $11.25 
2115  851         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 15.7
--------------------------------------
852
       ID  quantity                     item_name item_price
2116  852         1                 Steak Burrito     $8.99 
2117  852         1  Chips and Fresh Tomato Salsa     $2.39 
Somme totale de la commande = 11.38
--------------------------------------
853
       ID  quantity            item_name item_price
2118  853         1        Steak Burrit

2335  930         1  Chicken Bowl    $11.25 
Somme totale de la commande = 22.5
--------------------------------------
931
       ID  quantity      item_name item_price
2336  931         1   Chicken Bowl     $8.75 
2337  931         1  Bottled Water     $1.50 
2338  931         1          Chips     $2.15 
Somme totale de la commande = 12.4
--------------------------------------
932
       ID  quantity     item_name item_price
2339  932         2  Chicken Bowl    $16.98 
Somme totale de la commande = 16.98
--------------------------------------
933
       ID  quantity      item_name item_price
2340  933         1     Steak Bowl    $11.75 
2341  933         1  Bottled Water     $1.50 
Somme totale de la commande = 13.25
--------------------------------------
934
       ID  quantity      item_name item_price
2342  934         1   Chicken Bowl     $8.75 
2343  934         2          Chips     $4.30 
2344  934         1  Bottled Water     $1.50 
Somme totale de la commande = 14.55
---------

2539  1007         1  Side of Chips     $1.69 
Somme totale de la commande = 11.77
--------------------------------------
1008
        ID  quantity          item_name item_price
2540  1008         1       Chicken Bowl     $8.75 
2541  1008         1              Chips     $2.15 
2542  1008         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 12.15
--------------------------------------
1009
        ID  quantity      item_name item_price
2543  1009         1   Chicken Bowl     $8.49 
2544  1009         1    Canned Soda     $1.09 
2545  1009         1  Bottled Water     $1.09 
2546  1009         1  Side of Chips     $1.69 
Somme totale de la commande = 12.36
--------------------------------------
1010
        ID  quantity                            item_name item_price
2547  1010         1                        Carnitas Bowl    $11.48 
2548  1010         1                         Chicken Bowl     $8.49 
2549  1010         1                 Chicken Crispy Tacos     $8.49

2734  1085         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 22.95
--------------------------------------
1086
        ID  quantity            item_name item_price
2735  1086         1      Chicken Burrito    $11.25 
2736  1086         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 15.7
--------------------------------------
1087
        ID  quantity        item_name item_price
2737  1087         1  Chicken Burrito    $11.25 
2738  1087         1     Chicken Bowl     $8.75 
2739  1087         1            Chips     $2.15 
Somme totale de la commande = 22.15
--------------------------------------
1088
        ID  quantity         item_name item_price
2740  1088         1  Steak Salad Bowl    $11.89 
2741  1088         1      Chicken Bowl    $11.25 
Somme totale de la commande = 23.14
--------------------------------------
1089
        ID  quantity                            item_name item_price
2742  1089         1                         Chicken B

2912  1157         1  Chicken Crispy Tacos     $8.75 
Somme totale de la commande = 20.0
--------------------------------------
1158
        ID  quantity      item_name item_price
2913  1158         1  Carnitas Bowl    $11.75 
2914  1158         1  Carnitas Bowl    $11.75 
Somme totale de la commande = 23.5
--------------------------------------
1159
        ID  quantity            item_name item_price
2915  1159         1         Chicken Bowl    $11.25 
2916  1159         1  Chips and Guacamole     $4.45 
2917  1159         1                Chips     $2.15 
Somme totale de la commande = 17.849999999999998
--------------------------------------
1160
        ID  quantity     item_name item_price
2918  1160         1  Chicken Bowl     $8.19 
2919  1160         1  Chicken Bowl    $10.58 
Somme totale de la commande = 18.77
--------------------------------------
1161
        ID  quantity            item_name item_price
2920  1161         1  Chips and Guacamole     $4.45 
2921  1161        

3093  1232         1   Chicken Bowl    $11.25 
Somme totale de la commande = 12.75
--------------------------------------
1233
        ID  quantity      item_name item_price
3094  1233         1  Steak Burrito    $11.48 
Somme totale de la commande = 11.48
--------------------------------------
1234
        ID  quantity    item_name item_price
3095  1234         1  Veggie Bowl    $11.25 
3096  1234         1        Chips     $2.15 
Somme totale de la commande = 13.4
--------------------------------------
1235
        ID  quantity         item_name item_price
3097  1235         1   Chicken Burrito     $8.75 
3098  1235         1  Steak Salad Bowl     $9.39 
Somme totale de la commande = 18.14
--------------------------------------
1236
        ID  quantity             item_name item_price
3099  1236         1  Chicken Crispy Tacos     $8.49 
3100  1236         1   Chips and Guacamole     $3.99 
Somme totale de la commande = 12.48
--------------------------------------
1237
        ID  q

3309  1326         1  Chicken Crispy Tacos     $8.49 
Somme totale de la commande = 17.48
--------------------------------------
1327
        ID  quantity           item_name item_price
3310  1327         1  Chicken Salad Bowl     $8.75 
3311  1327         1        Chicken Bowl     $8.75 
3312  1327         1   Canned Soft Drink     $1.25 
Somme totale de la commande = 18.75
--------------------------------------
1328
        ID  quantity      item_name item_price
3313  1328         1  Steak Burrito    $11.75 
3314  1328         3  Bottled Water     $4.50 
Somme totale de la commande = 16.25
--------------------------------------
1329
        ID  quantity           item_name item_price
3315  1329         1  Chicken Salad Bowl     $8.75 
3316  1329         1     Chicken Burrito     $8.75 
Somme totale de la commande = 17.5
--------------------------------------
1330
        ID  quantity                              item_name item_price
3317  1330         1                    Chips and G

3502  1406         1    Steak Salad     $8.69 
Somme totale de la commande = 17.38
--------------------------------------
1407
        ID  quantity                     item_name item_price
3503  1407         1            Steak Crispy Tacos     $9.25 
3504  1407         1  Chips and Fresh Tomato Salsa     $2.95 
Somme totale de la commande = 12.2
--------------------------------------
1408
        ID  quantity                     item_name item_price
3505  1408         1                  Chicken Bowl    $10.98 
3506  1408         1  Chips and Fresh Tomato Salsa     $2.39 
Somme totale de la commande = 13.370000000000001
--------------------------------------
1409
        ID  quantity           item_name item_price
3507  1409         2  Chicken Salad Bowl    $22.50 
Somme totale de la commande = 22.5
--------------------------------------
1410
        ID  quantity     item_name item_price
3508  1410         2  Chicken Bowl    $21.96 
Somme totale de la commande = 21.96
------------------

1485
        ID  quantity      item_name item_price
3717  1485         1   Chicken Bowl     $8.49 
3718  1485         1    Canned Soda     $1.09 
3719  1485         1  Side of Chips     $1.69 
Somme totale de la commande = 11.27
--------------------------------------
1486
        ID  quantity          item_name item_price
3720  1486         1       Chicken Bowl    $11.25 
3721  1486         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 12.5
--------------------------------------
1487
        ID  quantity      item_name item_price
3722  1487         1   Chicken Bowl     $8.75 
3723  1487         1          Chips     $2.15 
3724  1487         1  Bottled Water     $1.50 
Somme totale de la commande = 12.4
--------------------------------------
1488
        ID  quantity      item_name item_price
3725  1488         1   Chicken Bowl     $8.49 
3726  1488         1  Side of Chips     $1.69 
Somme totale de la commande = 10.18
--------------------------------------
1489
       

3886  1558         1  6 Pack Soft Drink     $6.49 
Somme totale de la commande = 15.24
--------------------------------------
1559
        ID  quantity              item_name item_price
3887  1559         8          Side of Chips    $13.52 
3888  1559         2     Chicken Soft Tacos    $16.98 
3889  1559         2      Veggie Soft Tacos    $16.98 
3890  1559         2  Carnitas Crispy Tacos    $17.98 
3891  1559         2   Chicken Crispy Tacos    $16.98 
Somme totale de la commande = 82.44000000000001
--------------------------------------
1560
        ID  quantity     item_name item_price
3892  1560         1  Chicken Bowl     $8.75 
3893  1560         1  Chicken Bowl     $8.75 
Somme totale de la commande = 17.5
--------------------------------------
1561
        ID  quantity     item_name item_price
3894  1561         1  Chicken Bowl    $11.25 
3895  1561         1  Chicken Bowl    $11.25 
Somme totale de la commande = 22.5
--------------------------------------
1562
        ID  q

4068  1628         1    Steak Burrito     $8.69 
Somme totale de la commande = 16.88
--------------------------------------
1629
        ID  quantity      item_name item_price
4069  1629         1  Bottled Water     $1.09 
4070  1629         1  Side of Chips     $1.69 
4071  1629         1   Chicken Bowl     $8.49 
Somme totale de la commande = 11.27
--------------------------------------
1630
        ID  quantity                     item_name item_price
4072  1630         1               Chicken Burrito     $8.75 
4073  1630         1  Chips and Fresh Tomato Salsa     $2.95 
4074  1630         1             Canned Soft Drink     $1.25 
Somme totale de la commande = 12.95
--------------------------------------
1631
        ID  quantity            item_name item_price
4075  1631         1         Chicken Bowl     $8.49 
4076  1631         1  Chips and Guacamole     $3.99 
Somme totale de la commande = 12.48
--------------------------------------
1632
        ID  quantity         item_na

4298  1714         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 22.45
--------------------------------------
1715
        ID  quantity            item_name item_price
4299  1715         1        Steak Burrito    $11.75 
4300  1715         1  Chips and Guacamole     $4.45 
4301  1715         1    Canned Soft Drink     $1.25 
Somme totale de la commande = 17.45
--------------------------------------
1716
        ID  quantity            item_name item_price
4302  1716         1  Chips and Guacamole     $4.45 
4303  1716         1           Steak Bowl    $11.75 
Somme totale de la commande = 16.2
--------------------------------------
1717
        ID  quantity     item_name item_price
4304  1717         1    Steak Bowl     $9.25 
4305  1717         1  Chicken Bowl    $11.25 
4306  1717         1         Chips     $2.15 
Somme totale de la commande = 22.65
--------------------------------------
1718
        ID  quantity   item_name item_price
4307  1718         2  Steak B

4539  1804         1  Chips and Guacamole     $4.45 
Somme totale de la commande = 14.45
--------------------------------------
1805
        ID  quantity          item_name item_price
4540  1805         1       Chicken Bowl    $11.25 
4541  1805         1  Veggie Salad Bowl     $8.75 
Somme totale de la commande = 20.0
--------------------------------------
1806
        ID  quantity          item_name item_price
4542  1806         1      Bottled Water     $1.50 
4543  1806         1    Chicken Burrito     $8.75 
4544  1806         1      Bottled Water     $1.50 
4545  1806         1  Canned Soft Drink     $1.25 
Somme totale de la commande = 13.0
--------------------------------------
1807
        ID  quantity         item_name item_price
4546  1807         1     Steak Burrito     $9.25 
4547  1807         1  Steak Salad Bowl     $9.39 
Somme totale de la commande = 18.64
--------------------------------------
1808
        ID  quantity            item_name item_price
4548  1808        

In [107]:
mean = sum(dicoOrder.values()) / len(dicoOrder)

print("Le chiffre d'affaire moyen par commande est de", mean, "$.")

Le chiffre d'affaire moyen par commande est de 18.81142857142869 $.
